### PR TITLE
Added User Schema

### PR DESCRIPTION
**Pull request by imrishabh18**
Are there any more details to add in the User Schema ??

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -7,14 +7,13 @@ const path = require('path');
 const engine = require('ejs-locals');
 
 const app = express();
-const config = require('./config');
+const config = require('./config/config.example');
```
Comment (adwaith007) [inline app.js]: never include a example file

File: models/users.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    email: {
+        type  :String,
+        required : true
+    },
+    password: {
+        type : String,
+        required : true;
+    },
+    date: {
```
Comment (adwaith007) [inline models/users.js]: change date to something like "createdon" , something meaningful.

**Comment by adwaith007** [comment]
also please make sure u pull rebase from master before giving pr, so that merge conflicts dont occur. fix all merge conflicts.

File: app.js (hunk 1 above)
Comment (imrishabh18) [reply app.js]: Yeah even i was thinking why it does have an example. Okay, i will change the file name.

File: models/users.js (hunk 2 above)
Comment (imrishabh18) [reply models/users.js]: What does "createdon" here mean ?

**Comment by imrishabh18** [comment]
@adwaith007 check this branch now.

**Pull request closed without merging**